internal/commands/internal/test.go

```diff
@@ -0,0 +1,120 @@
+package internal
+
+import (
+	"errors"
+	"fmt"
+	"github.com/cirruslabs/cirrus-cli/internal/commands/logs"
+	"github.com/cirruslabs/cirrus-cli/pkg/larker"
+	"github.com/cirruslabs/cirrus-cli/pkg/larker/fs/local"
+	"github.com/cirruslabs/echelon/renderers"
+	"github.com/go-test/deep"
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var ErrTest = errors.New("test failed")
+
+var output string
+
+func test(cmd *cobra.Command, args []string) error {
+	// https://github.com/spf13/cobra/issues/340#issuecomment-374617413
+	cmd.SilenceUsage = true
+
+	// Discover tests
+	var testDirs []string
+	err := filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
+		// Does it look like a Starlark test?
+		if info.Name() == ".cirrus.expected.yml" {
+			testDirs = append(testDirs, filepath.Dir(path))
+		}
+
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+
+	// Configure hierarchical progress renderer
+	renderer := renderers.NewInteractiveRenderer(os.Stdout, nil)
```

Comment:
I don't think you need this here since you are using `logs.GetLogger` in the next lines.

Reply:
Good catch, removed in https://github.com/cirruslabs/cirrus-cli/pull/115/commits/670898f66e88fa8290b606b097ccd624b7027ca9.